+ docker image scan with layers

samq-ws opened the pull request.

tidharm reviewed (changes requested):
1. It would be a good idea to place the value of WS_DOCKER_INCLUDES (especially since there are ".*") within double quotes, especially for Ubuntu.

2. The "-d ./" in the execution command is redundant. When -d is not specified, the UA will scan the current directory.

3. It'd be a good habit to enclose the value of the -product and -project parameters within double quotes, especially when you're using Azure DevOps built-in variables for value.

4. Any particular reason you're using docker.includes and docker.layers as env variables, but docker.scanImages as CLI arg?

samq-ws commented:
Added double-quotes for bullets 1 & 3 and removed -d ./ for bullet 2.

I always use CLI args when available, and show env variables instead of creating a config file for when [CLI args do not exist](https://whitesource.atlassian.net/wiki/spaces/WD/pages/1544880156/Unified+Agent+Configuration+Parameters#Docker-Images).
